File: app/src/main/java/ardjomand/leonardo/arcmovies/model/Genres.java
package ardjomand.leonardo.arcmovies.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by unity on 04/03/18.
 */

public class Genres {

    @SerializedName("genres")
    @Expose
    private List<Genre> genres = null;

    public List<Genre> getGenres() {
        return genres;
    }

    public void setGenres(List<Genre> genres) {
        this.genres = genres;
    }
}

File: app/src/main/java/ardjomand/leonardo/arcmovies/model/Genres.kt
package ardjomand.leonardo.arcmovies.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by unity on 04/03/18.
 */

data class Genres(

        @SerializedName("genres")
        @Expose
        var genres: List<Genre>? = null
)
